tests/qa_automation/qa_run.pm

```diff
@@ -50,7 +50,12 @@ sub system_status {
 sub system_login {
     my $self = shift;
     $self->wait_boot;
-    select_console('root-console');
+    if (get_var('VIRTIO_CONSOLE')) {
+        select_console('root-virtio-terminal');
+    }
+    else {
+        select_console('root-console');
+    }
```

Comment:
I guess that is better handled more low-level. Based on the variable better set a root console variable. Or do you really nowhere else again need to select the virtio console then?